Plugins/Player/NWScript/nwnx_player.nss

```diff
@@ -57,16 +52,6 @@ void NWNX_Player_SetQuickBarSlot(object player, int slot, struct NWNX_Player_Qui
 // Get the name of the .bic file associated with the player's character.
 string NWNX_Player_GetBicFileName(object player);
 
-// Overrides the default visibility rules about how player perceives the target object.
-// NWNX_PLAYER_VISIBILITY_DEFAULT - Restore normal behavior
-// NWNX_PLAYER_VISIBILITY_HIDDEN - Object is always hidden from the player
-// NWNX_PLAYER_VISIBILITY_VISIBLE - Object is always shown to the player
-void NWNX_Player_SetVisibilityOverride(object player, object target, int override);
```

Comment:
Can't do this. Well, can, but shouldn't. Leave the old functions (and constants) in place, and make them call the new plugin instead, along with a deprecation warning to the log.